### ISSUE TITLE
itemOptions doesn't work

### ISSUE THREAD

**Issue by tongtoanbs** [yiisoft/yii2-bootstrap#81]
This code works if I use ```yii\widgets\ActiveForm``` instead of ```yii\bootstrap\ActiveForm```

```php
    <?= $form->field($model, 'type')->radioList(Customer::getTypes(), [
        'itemOptions' => [
            'data-attribute' => 'test'
        ]
    ]) ?>
```

**Comment by mikehaertl** [comment]
The above change should fix your issue. Maybe you want to confirm that it works for you? Note that it isn't merged into master yet, so you either have to pull my branch or apply the commit changes manually.

**Comment by mikehaertl** [comment]
Additional note: Please ignore commit d13fe14. It was a mistake and I've undone it again. Sorry for any confusion.

**Comment by klimov-paul** [comment]
Resolved by #82
